docs/versioned_docs/version-2.11.0/deprecated/releases-1x.md

```diff
@@ -1,6 +1,13 @@
 ---
 id: releases-1x
 title: OpenEBS 1.x Deprecated Releases
+keywords: 
+ - Change for OpenEBS 1.x
+ - Alpha features for OpenEBS 1.x
+ - Upgrade steps for OpenEBS 1.x
+ - Release notes for OpenEBS 1.x
+description: A release notes for all the versions of OpenEBS 1.X, which contains the change summary, alpha features and upgrade steps.
```

Comment:
```suggestion
description: A release notes for all the versions of OpenEBS 1.x, which contains the change summary, alpha features and upgrade steps.
```
Since it is in lowercase in other places